videoCallingQuickstart/app/src/main/java/com/example/videocallingquickstart/MainActivity.java

```diff
@@ -0,0 +1,313 @@
+            videoStreams[0] = currentVideoStream;
+            VideoOptions videoOptions = new VideoOptions(videoStreams);
+            options.setVideoOptions(videoOptions);
+            showPreview(currentVideoStream);
+        }
+        participants.add(new CommunicationUserIdentifier(calleeId));
+
+        call = callAgent.startCall(
+                context,
+                participants,
+                options);
+        call.addOnRemoteParticipantsUpdatedListener(this::handleRemoteParticipantsUpdate);
+        call.addOnStateChangedListener(this::handleCallOnStateChanged);
+    }
+
+    private void hangUp() {
+        try {
+            call.hangUp().get();
+        } catch (ExecutionException | InterruptedException e) {
+            e.printStackTrace();
+        }
+        if (previewRenderer != null) {
+            previewRenderer.dispose();
+        }
+    }
+
+    public void turnOnLocalVideo() {
+        List<VideoDeviceInfo> cameras = deviceManager.getCameras();
+        if(!cameras.isEmpty()) {
+            try {
+                VideoDeviceInfo cameraToUse = chooseCamera(cameras);
+                currentVideoStream = new LocalVideoStream(cameraToUse, this);
+                showPreview(currentVideoStream);
+                call.startVideo(this, currentVideoStream).get();
+            } catch (CallingCommunicationException acsException) {
+                acsException.printStackTrace();
+            } catch (ExecutionException | InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public void turnOffLocalVideo() {
+        try {
+            ((LinearLayout) findViewById(R.id.localvideocontainer)).removeAllViews();
+            previewRenderer.dispose();
+            previewRenderer = null;
+            call.stopVideo(this, currentVideoStream).get();
+        } catch (CallingCommunicationException acsException) {
+            acsException.printStackTrace();
+        } catch (ExecutionException | InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    VideoDeviceInfo chooseCamera(List<VideoDeviceInfo> cameras) {
+        for (VideoDeviceInfo camera : cameras) {
+            if (camera.getCameraFacing() == CameraFacing.FRONT) {
+                return camera;
+            }
+        }
+        return cameras.get(0);
+    }
+
+    private void showPreview(LocalVideoStream stream) {
+        // Create renderer
+        previewRenderer = new VideoStreamRenderer(stream, this);
+        LinearLayout layout = ((LinearLayout)findViewById(R.id.localvideocontainer));
+        preview = previewRenderer.createView(new CreateViewOptions(ScalingMode.FIT));
+        runOnUiThread(() -> {
+            layout.addView(preview);
+        });
+    }
+
+    private void handleCallOnStateChanged(PropertyChangedEvent args) {
+        if (call.getState() == CallState.CONNECTED) {
+            handleCallState();
```

Comment:
We should actually call this sumtin else or If we're not using this method anywhere else, we can move the 2 lines within it here and get rid of the method.